models/categories.models.js

```diff
@@ -85,3 +85,13 @@ DESC;
 
 
 }
+
+exports.addComment = (newComment, review_id) => {
+    //needs to be an author that exists for happy path
+    //need to handle if author doesnt exist
+    //need to handle when review_id doesnt exist
+    return db.query(`INSERT INTO comments(author, body, review_id) VALUES ($1, $2, $3) RETURNING *;`, [newComment.username, newComment.body, review_id]).then((result) => {
+        return result.rows[0]
```

Comment:
I've seen you destructure rows instead of having result.rows before. Please stay consistent and stick to one. Same with destruction your body in you test file